examples/to_ssa.py

```diff
@@ -105,6 +106,28 @@ def insert_phis(blocks, phi_args, phi_dests, types):
             instrs.insert(0, phi)
 
 
+def undefined_value(var_type):
+    values = {
+        "int": 0,
+        "bool": False,
+        "float": 0.0,
+    }
+    return values[var_type]
+
+
+def insert_undefined_vars(blocks, types):
+    entry = list(blocks.keys())[0]
+    types = sorted(set(types.values()))
```

Comment:
I believe this will crash on some of the benchmarks that use pointers. For example,
https://github.com/sampsyo/bril/blob/dbbd8e8f1d7000fefdcffaef2d79ce3339e28b40/benchmarks/eight-queens.bril#L6
` bril2json < ../../../benchmarks/eight-queens.bril | python ../../to_ssa.py`
The pointer type is stored in a `dict`, and thus unhashable.

Reply:
Yes, is this something that `to_ssa.py` should support? If so I can add it. There currently isn't a `null` value for pointers, but I suppose `alloc 0;` would work, and just never `free`.